reference/v8.json

```diff
@@ -826,32 +786,31 @@
       "mix"
     ]
   },
-  "function": {
+  "function-v7": {
```

Comment:
Yes, but when validating a v7 style the `reference` that will be passed in [here](https://github.com/mapbox/mapbox-gl-style-spec/blob/696fda69a24f59ef578db47558840404712b5895/lib/validate/parsed.js#L6) will come from `reference/v7.json`. [See here](https://github.com/mapbox/mapbox-gl-style-spec/blob/696fda69a24f59ef578db47558840404712b5895/lib/validate.js#L41).